### DISCUSSION TITLE
JWEDecryptionFailed: decryption operation failed

### DISCUSSION THREAD

**Discussion by memcaliber** [noam-honig/radweb#343] [Q&A]
Hi… I’ve just started learning Remult. I was previously familiar with Next.js. When I try to run the next-app-router-todo example in development mode, I encounter an error in the VS Code console. The error is related to [next-auth][error][JWT_SESSION_ERROR], with the message: ‘decryption operation failed’. I’ve followed the steps in the Readme.md sequentially, but it seems that the NEXTAUTH_SECRET is not being recognized. What steps should I take to ensure that the example application runs as expected?

- env.local already created with value like described on Readme.

**Comment by noam-honig** [comment]
Hi @memcaliber, can you share what was the problem and how you resolved it?

**Reply by memcaliber** [reply]
Hi .. My problem actually is i always get undefined value from env.process.NAME_OF_KEY. 

Problem Solving:
Actually, I do not recommend this method, considering that I am a beginner. I adopt this approach to facilitate my learning process. I add this inside authOptions, before 'provider':
 `secret: env.process.NAME_OF_KEY || "my-secret-keyword"`. This method is not dynamic, so please understand. 🙂
